artifactory/commands/transfer/settings.go

```diff
@@ -0,0 +1,43 @@
+package transfer
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/ioutils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strconv"
+)
+
+type TransferSettingsCommand struct {
+}
+
+func NewTransferSettingsCommand() *TransferSettingsCommand {
+	return &TransferSettingsCommand{}
+}
+
+func (tst *TransferSettingsCommand) Run() error {
+	var threadsNumberInput string
+	ioutils.ScanFromConsole("Choose the threads number", &threadsNumberInput, "")
+	threadsNumber, err := strconv.Atoi(threadsNumberInput)
+	if err != nil || threadsNumber < 1 {
+		return errorutils.CheckError(errors.New("the threads number must be a numeric positive value"))
```

Comment:
the threads number must be a numeric positive value
-->
The value must be a number between 1 and 100